code/modules/chemistry/tools/beakers.dm

```diff
@@ -11,10 +11,21 @@
 	icon_state = "beaker"
 	item_state = "beaker"
 	initial_volume = 50
-	var/image/fluid_image
 	var/icon_style = "beaker"
 	rc_flags = RC_SCALE | RC_VISIBLE | RC_SPECTRO
 	object_flags = NO_GHOSTCRITTER
+	var/solid_volume
+	var/liquid_volume
+	var/gas_volume
+	var/datum/reagents/solids
+	var/datum/reagents/liquids
+	var/datum/reagents/gases
+	var/solid_fluid_state
+	var/liquid_fluid_state
+	var/image/solid_fluid_image
+	var/image/liquid_fluid_image
+	var/image/gas_fluid_image
```

Comment:
why are these vars here and not local to the `update_icon()`, since they appear to always be getting nulled